src/core/output-layout.cpp

```diff
@@ -1387,6 +1388,17 @@ class output_layout_t::impl
             }
         }
 
+        /* Disable DPMS state outputs */
+        for (auto& entry : config)
+        {
+            auto& handle = entry.first;
+            auto& state  = entry.second;
+            if (state.source & OUTPUT_IMAGE_SOURCE_DPMS)
+            {
+                wlr_output_enable(handle, false);
```

Comment:
`wlr_output` state is double-buffered, so you need a `wlr_output_commit` after this. In general though, I do not see why you need this.